challenges/functions.js

```diff
@@ -15,6 +19,22 @@
 */
 
 
+function add (param1, param2){
+  return (param1 + param2);
+}
+consume (2, 3, add);
+
+
+function multiply (param1, param2,){
+  return (param1 * param2);
+}
+consume(2, 3, multiply);
+
+function greeting (first_name, last_name){
+  return (`Hello ${first_name} ${last_name}, nice to meet you!`);
```

Comment:
`( `Hello ${first_name} ${last_name}, nice to meet you!` )`;
The outer parenthesis are unnecessary.